Source/rclUE/Private/ROS2Node.cpp

```diff
@@ -167,55 +167,67 @@ void AROS2Node::AddService(const FString ServiceName, const TSubclassOf<UROS2Gen
     }
 }
 
-void AROS2Node::AddPublisher(UROS2Publisher* Publisher)
+void AROS2Node::AddPublisher(UROS2Publisher* InPublisher)
 {
-    check(IsValid(Publisher));
+    check(IsValid(InPublisher));
 
-    if (!Publisher->UpdateDelegate.IsBound())
+    if (!InPublisher->UpdateDelegate.IsBound())
     {
         UE_LOG(LogROS2Node, Warning, TEXT("UpdateDelegate is not set - is this on purpose? (%s)"), *__LOG_INFO__);
     }
 
-    Publisher->RegisterComponent();
-    Publisher->OwnerNode = this;
-    Publishers.Add(Publisher);
+    if (false == Publishers.Contains(InPublisher))
```

Comment:
Just curious : are there reasons to use `false == ...` instead of `! ...` like in if statements above ?

Reply:
Ah there is not a specific reason of this (and also the Yoda style in general), it had just become my preference after inadvertently missing `!` once, since then I've been using it to purposefully require attention on certain places.